index.html

```diff
@@ -1125,185 +1228,6 @@ <h2>
         </dd>
       </dl>
     </section>
-    <h2 id="protocol-registry">
-      Registry of protocols
```

Comment:
I agree we should delete this, but I'm wondering if any of this is salvageable for how new protocols are added to the spec. Should we have a non-normative note about "file a bug if you think your protocol belongs here?" or something?